src/nuxeo-cloud/day-to-day/support.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Support and Maintenance
+review:
+    comment: ''
+    date: '19-03-2020'
+    status: ok
+labels:
+    - nuxeo-cloud
+tree_item_index: 500
+toc: true
+---
+
+## Support
+
+Nuxeo is available to help support issues that come up after the environments have been provisioned and start being used. Typical issues include:
```

Comment:
change this section to 

Nuxeo Support and Cloud teams are available to support issues that arise as teams begin deploying changes and using the platform. Typical Nuxeo Cloud support requests include:
- Data restoration from Production to Pre-Production
- Deployment of Marketplace Packages, 3rd party services, or Configuration changes

The Nuxeo support team provides first level support to evaluate all issues and route to the appropriate teams, including Nuxeo Cloud.

Nuxeo is available 24x7 to support site availability issues. If other issues are found, the team will address them following our standard customer support SLA’s depending on subscription level.